src/drive/web/modules/filelist/FilenameInput.jsx

```diff
@@ -65,22 +63,30 @@ export default class FilenameInput extends Component {
   abort(accidental = false) {
     this.props.onAbort && this.props.onAbort(accidental)
   }
-
+  handleFocus() {
+    const { name } = this.props
+    const { filename } = CozyFile.splitFilename({ name, type: 'file' })
+    //Since we're mounting the component and focusing it at the same time
+    // let's add a small timeout to be sure the ref is populated
+    setTimeout(() => {
+      this.textInput.current.setSelectionRange(0, filename.length)
+    }, 5)
```

Comment:
This is a bit meh, 5ms is really arbitrary. `useEffect` is cool for things like that but would require "unclassing" the component.